pkgs/tools/networking/libreswan/default.nix

```diff
@@ -51,8 +58,6 @@ stdenv.mkDerivation {
     sed -i -e 's|#!/usr/bin/python|#!/usr/bin/env python|' -e 's/^\(\W*\)installstartcheck()/\1sscmd = "ss"\n\0/' programs/verify/verify.in
   '';
 
-  patches = [ ./libreswan-3.18-glibc-2.26.patch ];
```

Comment:
Should this patch file be removed from the tree?

Reply:
Good catch. Removed.